fury/io.py

```diff
@@ -24,6 +25,13 @@ def load_image(filename, as_vtktype=False, use_pillow=True):
         desired image array
 
     """
+    is_url = filename.startswith('http://') or filename.startswith('https://')
```

Comment:
`filename.lower().startswith('http://')` in case you have `HTTP://`

Reply:
Added the tests/requested changes.